decidim-core/spec/presenters/decidim/notification_presenter_spec.rb

```diff
@@ -58,6 +58,23 @@ module Decidim
           end
         end
       end
+
+      describe "#display_resource_text?" do
+        it "returns false if the notification hasn't to display the content of the comment" do
+          expect(subject.display_resource_text?).to eq(false)
+        end
+      end
+    end
+
+    context "with a valid comment notification" do
+      let(:notification) { create(:notification, :comment_notification) }
```

Comment:
If we drop the `context "with comment notifications"` from `decidim-core/spec/system/notifications_spec.rb` then we should refactor this, so the trait is defined here as it's the only place that needs it.

Reply:
👌